src/test/warning/warning-printer_test.ts

```diff
@@ -49,7 +60,7 @@ suite('WarningPrinter', () => {
 
   test('can handle printing no warnings', async() => {
     await printer.printWarnings([]);
-    assert.equal(output.toString(), '');
+    assert.deepEqual(output.toString(), '');
```

Comment:
They're semantically identical, but the output is *way* more legible for multiline strings, as it will display a diff of only the lines that didn't match, so I think deepEqual is the better default.